backend/src/test/java/ai/verta/modeldb/HydratedServiceTest.java

```diff
@@ -4052,11 +4056,6 @@ public void findHydratedDatasetsTest() {
         isExpectedResultFound = true;
         for (HydratedDataset hydratedDataset : response.getHydratedDatasetsList()) {
           Dataset dataset = hydratedDataset.getDataset();
-          assertEquals(
```

Comment:
because of the update cron the time stamp is different and the compare fails . Important parts of the objects are compared below